cms/templates/howitworks.html

```diff
@@ -146,7 +162,7 @@ <h2 class="sr">${_("Sign Up for {studio_name} Today!").format(studio_name=settin
 
     <ul class="list-actions">
       <li class="action-item">
-        <a href="${reverse('signup')}" class="action action-primary">${_("Sign Up &amp; Start Making an {platform_name} Course").format(platform_name=settings.PLATFORM_NAME)}</a>
+        <a href="${reverse('signup')}" class="action action-primary">${_("Sign Up & Start Making an {platform_name} Course").format(platform_name=settings.PLATFORM_NAME)}</a>
```

Comment:
We don't have a way to handle that. The best we can do is avoid the article, "Start Making Your {platform_name} Course", but of course, there's nothing ensuring that a translation of that string won't have the same problem as the article does in English.